requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 -e .
+coverage
 flake8
 pytest
-pytest-coverage
```

Comment:
I don't know why but `pytest-cov` stopped producing XML coverage reports. Despite what it's logs said, a `coverage.xml` file just wasn't produced. (And FWIW, the package name is `pytest-cov`, not `pytest-coverage` or `pytest-cover`.